src/Microsoft.TemplateEngine.Edge/Settings/ComponentManager.cs

```diff
@@ -242,7 +224,9 @@ private bool RegisterType(Type type)
             }
         }
 
-        private void Save()
+#pragma warning disable SA1202 // Elements should be ordered by access, disabled to make review easier
```

Comment:
I vote to disable the rule, @bekir-ozturk thoughts?

Reply:
Another problem with this rule is that if you don't have additional tools installed in VS, its hard to maintain this orders and move methods around as they change...